### PR TITLE
Fixed issue#72

### PR DESCRIPTION
**Pull request by luojiebin**
Support specifying timeout values using strings instead of ints.
Timeout can be: 1, '1s', '1S', '1m' ,'1M', '1h', '1H'.

### CODE REVIEW

**Comment by luojiebin** [comment]
Is this PR ok to merged? Or anything I can do to improve it so that it can be merged?

File: rq/utils.py [hunk 1]
```diff
@@ -242,3 +244,21 @@ def backend_class(holder, default_name, override=None):
         return import_attribute(override)
     else:
         return override
+
+
+def transfer_timeout(timeout):
```
Comment (selwin) [inline rq/utils.py]: Could you please rename this function to `parse_timeout`?

Comment (luojiebin) [inline rq/utils.py]: @selwin Updated it. Thanks

**Comment by selwin** [comment]
Thanks! Could you please document this in `https://python-rq.org` so people can discover this feature? The documentation lives in the `gh-pages` branch.

**Comment by luojiebin** [comment]
@selwin happy to do it

**Comment by luojiebin** [comment]
@selwin Hi, it's really strange that I can't see my commits in https://github.com/nvie/rq/graphs/contributors. Is it a bug in Github?

**Comment by selwin** [comment]
Hmm I'm not sure. It's also possible that the graph is not generated in
real time.


Win~

**Pull request merged**